infra/jupyterhub/templates/tljh_config.yml.j2

```diff
@@ -22,3 +22,7 @@ auth:
   GoogleOAuthenticator:
     client_id: {{ google_auth_client_id }}
     client_secret: {{ google_auth_client_secret }}
+services:
+  cull:
```

Comment:
c'est pas cull-idle le service ?

Reply:
Si, mais dans la conf de The Little Jupyterhub, ils l'appellent comme ça.
http://tljh.jupyter.org/en/latest/topic/idle-culler.html
En faisant 
sudo tljh-config set services.cull.timeout 6000
sudo tljh-config set services.cull.every 600
sudo tljh-config reload
La config se modifie de cette façon automatiquement

Reply:
👍